src/sdk/pynni/nni/compression/torch/builtin_pruners.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, model, config_list):
             - sparsity
         """
         super().__init__(model, config_list)
-        self.mask_list = {}
+        self.mask_dict = {}
```

Comment:
`self.mask_dict` already exists in `Pruner` class

Reply:
@QuanluZhang fixed.